<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Log</string>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lned_log_label"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Variable</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cbbx_logging_variable"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Label</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Pen</string>
           </property>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spbx_pen_width"/>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbbx_pen_color">
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Orange</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yelloish-Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Greenish-Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blueish-Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pink</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lned_pen_color">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Brush</string>
           </property>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="2" column="1">
             <widget class="QLineEdit" name="lned_brush_color">
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbbx_brush_color">
              <item>
               <property name="text">
                <string>Red</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Orange</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yellow</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Yelloish-Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Green</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Greenish-Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blueish-Cyan</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Blue</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Purple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Magenta</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Pink</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Transparency</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="spbx_brush_transparency"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Axes</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Horizontal axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QRadioButton" name="rdbt_horizontal_axis_ontop">
              <property name="text">
               <string>Top</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rdbt_horizontal_axis_onbottom">
              <property name="text">
               <string>Bottom</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Vertical axis</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="rdbt_vertical_axis_onright">
              <property name="text">
               <string>Right</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rdbt_vertical_axis_onleft">
              <property name="text">
               <string>Left</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
